locale.c

```diff
@@ -441,6 +441,22 @@ S_wsetlocale(const int category, const wchar_t * wlocale)
 #  endif
 #endif  /* WIN32_USE_FAKE_OLD_MINGW_LOCALES */
 
+/* 'for' loop headers to hide the necessary casts */
```

Comment:
Since these are only usable inside the control block of a `for` loop, I think it makes more sense to include that in the macro definition, e.g. 

```
#define for_all_individual_category_indexes(i) \
    for (locale_category_index i = … 
```

Reply:
Done